dev-cpp/sourcetrail/sourcetrail-2018.3.13.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop gnome2-utils xdg-utils
+
+DESCRIPTION="A cross-platform source explorer for C/C++ and Java"
+HOMEPAGE="https://www.sourcetrail.com/"
+SRC_URI="https://www.sourcetrail.com/downloads/${PV}/linux/64bit -> ${P}.tar.gz"
+
+LICENSE="Sourcetrail"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="examples"
+
+DEPEND="dev-util/patchelf"
+
+S="${WORKDIR}/Sourcetrail"
+QA_PREBUILT="*"
+
+src_install() {
+	# This removes the rpath entries with $$ORIGIN
+	# since they trigger warnings when merging
+	for f in "Sourcetrail" "sourcetrail_indexer"; do
+		rpath=$(patchelf --print-rpath "${f}" 2>/dev/null)
+		new_rpath=${rpath//\$\$ORIGIN\/lib\/:/}
+		patchelf --set-rpath "${new_rpath}" "${f}" || die
+	done
+
+	insinto /opt/sourcetrail
+	doins -r EULA.txt README data lib plugin
+	if use examples; then
+		doins -r user
+	fi
+	exeinto /opt/sourcetrail
+	doexe Sourcetrail sourcetrail_indexer Sourcetrail.sh resetPreferences.sh
+	dodir /usr/bin
+	newbin - sourcetrail <<-EOF
```

Comment:
Hmm, to be honest I think it belongs in `/opt/bin`. You can use `into /opt` to make `newbin` install there.

Reply:
I changed the ebuild so that the script is installed in /opt/bin and then modified the .desktop file to use that.